### PR TITLE
shutter: Add version 4.4

### PR DESCRIPTION
**Pull request by wenmin92**

### CODE REVIEW

File: bucket/shutter.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "https://www.den4b.com/download/shutter/portable/4.4?token=scoop#/dl.zip",
+    "hash": "a2cc77e2ecbc126398723aa57dfe00e653747d0997b01cc0d7ebb04f78162fb3",
+    "bin": "Shutter.exe",
+    "shortcuts": [
+        [
+            "Shutter.exe",
+            "Shutter"
+        ]
+    ],
+    "pre_install": "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item -Path \"$dir\\Settings.ini\" -ItemType 'File' | Out-Null }",
```
Comment (Ash258) [inline bucket/shutter.json:18]: - Installers are always before binaries and shortcuts
- Do not use `-ItemType` for files and `-Path` in general for basic cmdlets

File: bucket/shutter.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://www.den4b.com/products/shutter",
+    "version": "4.4",
+    "license": {
+        "identifier": "CC-BY-NC-ND-3.0",
+        "url": "https://www.den4b.com/license"
+    },
+    "description": "A multifunctional scheduling utility, which has a user friendly and easy-to-use interface and supports many different Events and Actions.",
```
Comment (Ash258) [inline bucket/shutter.json:8]:
```suggestion
    "version": "4.4",
    "description": "A multifunctional scheduling utility, which has a user friendly, easy-to-use interface and supports many different Events and Actions.",
    "homepage": "https://www.den4b.com/products/shutter",
    "license": {
        "identifier": "CC-BY-NC-ND-3.0",
        "url": "https://www.den4b.com/license"
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Persist presets

A

File: bucket/shutter.json (hunk 1 above)
Comment (wenmin92) [reply bucket/shutter.json]: Thanks!

**Comment by wenmin92** [comment]
> Persist presets
> 
> A

Sorry, I don't quite understand what this means.

**Comment by Ash258** [comment]
Persist presets folder

**Review by Ash258 — APPROVED** [review]

**Comment by Ash258** [comment]
@wenmin92  Are you drunk or you just enjoy wasting my time??

A

**Comment by wenmin92** [comment]
@Ash258 I'm sorry for the trouble, I'm not very familiar with workflow.🙏🙏🙏 
I think if you had approved, the `package-fix-needed` label should be removed. 
Sorry, I will be more patient.

**Pull request merged**